.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-18.04
```

Comment:
* This is an example of an inline comment. 🙂

You can see how it's under the main summary above and point to a specific part of the code where you want to suggest changes or just comment on it generally. 😉

To do this, make sure you go to the "File changes" tab and use the "Start a review" button for your first review comment.